_data/simple-icons.json

```diff
@@ -475,11 +525,21 @@
             "hex": "2A73CC",
             "source": "https://about.coursera.org/press"
         },
+        {
+            "title": "Creative Commons",
+            "hex": "FFFFFF",
+            "source": "https://creativecommons.org/use-remix/made-with-cc/"
```

Comment:
Would prefer just https://creativecommons.org/, for the same reasons as mentioned on BlackBerry